docs/source/introduction/quickstart.ipynb

```diff
@@ -22,45 +22,56 @@
     "\n",
```

Comment:
<p>full stop missing it seems.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probabilistic-numerics/probnum/pull/419/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>done</p>